deploy/deployment/openshift/objects/che-operator.ClusterRoleBinding.yaml

```diff
@@ -0,0 +1,29 @@
+#
```

Comment:
Does it makes sense to use kustomize to not to duplicate for Kubernetes and Openshift?